MyCryptoHQ/MyCrypto
# Update copy / links for Beta

Closes #1165

### Description

Updates various parts of the site to present a more confident and friendly beta. Also links to our new discord server! The copy is by no means final, and the prevalence of Discord links was something I thought would be better than support emails, so feedback is welcome.

### Changes

* Update `AlphaAgreement` to `BetaAgreement`, and update the copy and button behavior to be more welcoming.
* Update the announcement banner copy and color. Now it points people to the repo and discord to file issues and discuss beta.
* Add discord link to the footer complete with icon.
* Update error screen to encourage filing a GitHub issue.

### Steps to Test

1. Run app on `beta-prep` branch
2. Load page, confirm beta agreement pops up
3. Click no, confirm you end up on production
4. Go back, click yes, confirm you end up on beta
5. Confirm announcement is updated, links work
6. Confirm footer is updated, links work
7. Reload page, confirm you don't get beta agreement again

### Screenshots


<img width="649" alt="screen shot 2018-02-27 at 1 59 11 pm" src="https://user-images.githubusercontent.com/649992/36749398-9fcbeec8-1bc8-11e8-9ee9-182af5c5f661.png">

<img width="1440" alt="screen shot 2018-02-27 at 1 57 51 pm" src="https://user-images.githubusercontent.com/649992/36749399-a2400ac2-1bc8-11e8-96bd-7bad277822c1.png">

<img width="287" alt="screen shot 2018-02-27 at 2 16 58 pm" src="https://user-images.githubusercontent.com/649992/36749482-e5f90eee-1bc8-11e8-9329-540cd8b93dee.png">
